### PR TITLE
Fix repo for SLES i586

### PR DESCRIPTION
**Pull request by eabecho**
Was pointing to non-existent i386 repo directory rather than i586.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -172,11 +172,19 @@
               }
             }
           }
+          case $::architecture {
+            /^i386/: {
+              $_architecture = 'i586'
```
Comment (emahags) [inline manifests/init.pp]: Please use architecture_real instead of _architecture to follow the rest of the module.

**Comment by emahags** [comment]
Good find. Could you add a test that fails before your fix and works afterwards?

**Comment by eabecho** [comment]
Added checks.  Discovered latest actually points to esx/5.0p12 rather than esx/6.0.

**Comment by emahags** [comment]
Great! Just do a rebase of these commits and I'll merge it.

**Pull request merged**